### PR TITLE
RPC Jurisdiction Service

### PR DESCRIPTION
**Pull request by drtyh2o**
## 📚 Purpose
Contribute RPC common module and convert jurisdiction service/API to RPC. 

Also, fix some missing dependencies and project references.

## 📦 Impacts:
- [x] mds-jurisdiction
- [x] mds-jurisdiction-service

### CODE REVIEW

File: packages/mds-rpc-common/tests/index.spec.ts [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    onStop: async () => undefined
+  },
+  {
+    length: async ([word]) =>
+      word && word.length > 0 ? ServiceResult(word.length) : ServiceError({ type: 'NotFoundError', message: 'No Word' })
+  }
+).controller()
+
+describe('Test RPC Client', () => {
+  const TestClient: ServiceClient<TestService> = {
```
Comment (janedotx) [inline packages/mds-rpc-common/tests/index.spec.ts:47]: I don't quite understand the purpose of this `length` property. It seems to me like it stores the service name and returns the length of the name if the service was initialized successfully. Am I right?

File: packages/mds-rpc-common/tests/index.spec.ts [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    length: word => RpcRequest(RpcClient(TestServiceRpcDefinition).length, [word])
```
Comment (janedotx) [inline packages/mds-rpc-common/tests/index.spec.ts:48]: What's this `[word]` thing? Some kind of destructuring?

Comment (drtyh2o) [reply packages/mds-rpc-common/tests/index.spec.ts:48]: Client method arguments always get passed to the rpc layer as an array.

File: packages/mds-rpc-common/tests/index.spec.ts (hunk 1 above)
Comment (drtyh2o) [reply packages/mds-rpc-common/tests/index.spec.ts:47]: It’s a silly test client method; length is a method that accepts a string, invokes the rpc method which returns its length.

Comment (janedotx) [reply packages/mds-rpc-common/tests/index.spec.ts:47]: Cool, could you add a comment to that effect, that it's purely arbitrary and there for testing?

**Pull request merged**